### PR TITLE
test: first draft of doc of hacking selenium and avocado tests

### PR DESCRIPTION
**Pull request by jscotka**

### CODE REVIEW

**Comment by jscotka** [comment]
@KKoukiou I found I've done there issue when commiting, I've by mistake had changed author of commit. If you know how to change it to me, please do so and we can merge this. I'm not so familiar with git and have fear that I've discard your commit when trying to play with my commit author :-)

**Comment by KKoukiou** [comment]
> @KKoukiou I found I've done there issue when commiting, I've by mistake had changed author of commit. If you know how to change it to me, please do so and we can merge this. I'm not so familiar with git and have fear that I've discard your commit when trying to play with my commit author :-)

Pull the changes from this remote branch, squash the two commits (there is squah option in the interactive rebase) and then do git commit --am and apply your new changes. Just try it locally, no worries if something goes wrong, if you don't push it you want lose my changes here.

**Comment by jscotka** [comment]
@KKoukiou squashing seems well, thanks.

**Comment by martinpitt** [comment]
Thank you, this is great! Can you please link to this README file from [test/README.md](https://github.com/cockpit-project/cockpit/blob/master/test/README.md#running-tests)? This currently also mentions how to run the avocado/selenium tests, but without much detail. Replacing this with a link would be best, I think.

**Comment by jscotka** [comment]
@martinpitt thanks for hint, makes sense, added link to main test README file.
Is that okay now?

File: test/README.md [hunk 1]
```diff
@@ -178,15 +178,12 @@ you run `test/containers/run-tests` you need to use the `-i` option to
 build/install cockpit into the test VM. This needs to be done with a compatible
 `TEST_OS` (usually a recent `fedora-*`).
 
+### Avocado and Selenium tests
 The third class of integration tests use avocado and selenium to cover
-different browsers:
+different browsers
```
Comment (KKoukiou) [inline test/README.md]: missing dot here, it's the end of a sentence.

File: test/README.md [hunk 2]
```diff
@@ -178,15 +178,12 @@ you run `test/containers/run-tests` you need to use the `-i` option to
...
 
-    $ bots/image-download selenium
-    $ bots/image-prepare fedora-29
-    $ TEST_OS=fedora-29 test/avocado/run-tests --selenium-tests --browser=firefox -v
+For details how run and debug see [selenium hacking guide](./avocado/README.md)
```
Comment (KKoukiou) [inline test/README.md]: For more details on how to run and debug these tests see ...

**Comment by jscotka** [comment]
@KKoukiou thanks, changes incorporated and commits squashed

**Pull request merged**